app/lib/fisbot/failure_group_api.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class FailureGroupApi < FisbotApi
+  class << self
+    def resource
+      :failure_groups
+    end
+
+    def failure_group_description(job_id)
+      uri = api_access.resource_uri(job_id.to_s)
+      array = JSON.parse(Net::HTTP.get(uri))
+      return array['description'] if array.present?
```

Comment:
Rename `array` to `response`. 
Once we change the route as suggested in the PR review for https://github.com/blackducksoftware/fisbot/pull/619/files, the line 10 will become:
```
uri = api_access.resource_uri("find_by_job/#{job_id}")